debian/libmiral7.symbols

```diff
@@ -143,7 +143,6 @@ libmiral.so.7 libmiral7 #MINVER#
  (c++)"miral::PrintTo(miral::Window const&, std::basic_ostream<char, std::char_traits<char> >*)@MIRAL_5.0" 5.0.0
  (c++)"miral::SessionLockListener::SessionLockListener(std::function<void ()> const&, std::function<void ()> const&)@MIRAL_5.0" 5.0.0
  (c++)"miral::SessionLockListener::operator()(mir::Server&) const@MIRAL_5.0" 5.0.0
- (c++)"miral::SessionLockListener::~SessionLockListener()@MIRAL_5.0" 5.0.0
```

Comment:
As discussed on Matrix, it is problematic to remove symbols that are part of a released ABI. (It is unclear what change led to these being reported missing)